modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java

```diff
@@ -6,14 +6,351 @@
+		super.testGetProductIdImagesPageWithPagination();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testGraphQLDeleteAttachment() throws Exception {
+		super.testGraphQLDeleteAttachment();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testGraphQLGetAttachmentByExternalReferenceCode()
+		throws Exception {
+
+		super.testGraphQLGetAttachmentByExternalReferenceCode();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testGraphQLGetAttachmentByExternalReferenceCodeNotFound()
+		throws Exception {
+
+		super.testGraphQLGetAttachmentByExternalReferenceCodeNotFound();
+	}
+
+	@Override
+	@Test
+	public void testPatchAttachmentByExternalReferenceCode() throws Exception {
+		_testPatchAttachmentByExternalReferenceCode();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductByExternalReferenceCodeAttachmentByBase64()
+		throws Exception {
+
+		super.testPostProductByExternalReferenceCodeAttachmentByBase64();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductByExternalReferenceCodeAttachmentByUrl()
+		throws Exception {
+
+		super.testPostProductByExternalReferenceCodeAttachmentByUrl();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductByExternalReferenceCodeImage() throws Exception {
+		super.testPostProductByExternalReferenceCodeImage();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductByExternalReferenceCodeImageByBase64()
+		throws Exception {
+
+		super.testPostProductByExternalReferenceCodeImageByBase64();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductByExternalReferenceCodeImageByUrl()
+		throws Exception {
+
+		super.testPostProductByExternalReferenceCodeImageByUrl();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductIdAttachmentByBase64() throws Exception {
+		super.testPostProductIdAttachmentByBase64();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductIdAttachmentByUrl() throws Exception {
+		super.testPostProductIdAttachmentByUrl();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductIdImage() throws Exception {
+		super.testPostProductIdImage();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductIdImageByBase64() throws Exception {
+		super.testPostProductIdImageByBase64();
+	}
+
+	@Ignore
+	@Override
+	@Test
+	public void testPostProductIdImageByUrl() throws Exception {
+		super.testPostProductIdImageByUrl();
+	}
+
+	@Override
+	protected Attachment randomAttachment() throws PortalException {
+		FileEntry fileEntry = _dlAppLocalService.addFileEntry(
+			null, TestPropsValues.getUserId(), testGroup.getGroupId(),
+			DLFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+			TempFileEntryUtil.getTempFileName(
+				RandomTestUtil.randomString() + ".txt"),
+			ContentTypes.TEXT_PLAIN, RandomTestUtil.randomString(),
+			StringPool.BLANK, StringPool.BLANK, StringPool.BLANK,
+			new ByteArrayInputStream(RandomTestUtil.randomBytes()), 0, null,
+			null, null, _serviceContext);
+
+		return new Attachment() {
```

Comment:
please add a method that links fileEntry to attachment by fileEntryERC, so you can prove that your endpoints work as expected.
Thank you!